tests/rust/tcp-test/close/mod.rs

```diff
@@ -7,7 +7,13 @@
 
 use anyhow::Result;
 use demikernel::{
-    runtime::fail::Fail,
+    runtime::{
+        fail::Fail,
+        types::demi_opcode_t::{
+            DEMI_OPC_CLOSE,
+            DEMI_OPC_FAILED,
+        },
```

Comment:
Rather then including these, could you use them in the code like `demi_opcode_t::DEMI_OPC_CLOSE` and `demi_opcode_t::DEMI_OPC_FAILED`? It cleans up the namespace.